azurerm/internal/services/containers/container_registry_data_source.go

```diff
@@ -96,10 +91,6 @@ func dataSourceContainerRegistryRead(d *schema.ResourceData, meta interface{}) e
 		d.Set("sku", string(sku.Tier))
 	}
 
-	if account := resp.StorageAccount; account != nil {
```

Comment:
and it could be an old account so maybe we should leave this till 3.0 - should be migrated isn't a 100% yes they are all migrated